android/pubsub.md

```diff
@@ -68,6 +68,17 @@ Then, you need to attach the `myIOTPolicy` policy to the user's *Cognito Identit
 aws iot attach-principal-policy --policy-name 'myIOTPolicy' --principal '<YOUR_COGNITO_IDENTITY_ID>'
 ```
 
+You can also programmatically attach the `myIOTPolicy` policy to the user's *Cognito Identity Id* as follows :
+
+```
+AttachPolicyRequest attachPolicyReq = new AttachPolicyRequest();
```

Comment:
Do we need to add any import statements?

Reply:
Added the required import statements

Reply:
please break this into two code blocks to make it clear that import statements have to be added to the top of the file